### PR TITLE
Issue #800 resolved

### LINKED ISSUE #800: Get WIFI Passwords
**This template is strictly for requesting New Scripts, if you want a new feature (automating this Repo,**
**a change in structure) in the Repo,** 
**or you want to report a Bug. There are other templates which you should follow**

# Description
A Python Script which when Executed on the target machine will send all the WIFI names and their respective passwords stored in it, through Given Email.

## Language
- [x] Python

## Checklist:

- **Details on the Script**

| name            | about                                                  | labels  | assignees                                                      |
| Get WIFI Passwords | A Python Script which when Executed on the target machine will send all the WIFI names and their respective passwords stored in it, through Given Email. |SWOC| @pritamp17 |

### PR DESCRIPTION
**Pull request by get-satyam**
# Description

A Python Script which when Executed on the target machine will set up an SMTP server and send all the WIFI names and their respective passwords stored in it, through Given Email.

Fixes #800

## Type of change

_Please delete options that are not relevant._

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works

### CODE REVIEW

File: Python/WIFI PASS/WIFI.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+#import necessary files
```
Comment (vybhav72954) [inline Python/WIFI PASS/WIFI.py:1]: Rename the Directory to

```WiFi_Passwrod```

and python files to smaller case

**Review by vybhav72954 — CHANGES_REQUESTED** [review]

File: Python/WIFI PASS/WIFI.py (hunk 1 above)
Comment (vybhav72954) [reply Python/WIFI PASS/WIFI.py:1]: Nevermind please don't make any changes

**Comment by vybhav72954** [comment]
The scripts have been plagiarised from [here](https://gist.github.com/brentvollebregt/30d278eae98e2ff221add008259d42bb) and [here](https://school.geekwall.in/p/BJb4hizyz).

These might not be the actual source but are a clear case of plag @HarshCasper 

Closing the PR. @get-satyam Feel free to make a new or, but with original script please

**Comment by get-satyam** [comment]
@vybhav72954 is the PR merged yet? or do I need to do the necessary changes?

**Comment by vybhav72954** [comment]
The PR has been closed (not merged) because of plagiarism, I cant merge a code that has been copied from someone else's work.

**Pull request closed without merging**